src/Og.php

```diff
@@ -17,6 +17,137 @@
 class Og {
 
   /**
+   * Static cache for groups per entity.
+   *
+   * @var array
+   */
+  protected static $entityGroupCache = [];
+
+  /**
+   * Create an organic groups field in a bundle.
+   *
+   * @param $field_name
+   *   The field name.
+   * @param $entity_type
+   *   The entity type.
+   * @param $bundle
+   *   The bundle name.
+   */
+  public static function createField($field_name, $entity_type, $bundle) {
```

Comment:
in that case we need to pass the $options we had [here](https://github.com/Gizra/og/blob/cc9c04036a80fcf27a4e9533b6cb0820e3e750be/og.module#L3331-L3334)